aim/tools/cli/commands/infra.py

```diff
@@ -56,3 +56,34 @@ def create(ctx):
     ctx.obj['apic_manager'].ensure_infra_created_on_apic()
     ctx.obj['apic_manager'].ensure_bgp_pod_policy_created_on_apic()
     ctx.obj['apic_manager'].ensure_opflex_client_cert_validation_disabled()
+
+
+@infra.command(name='tag-list')
+@click.option('--tag-name', '-t', help='Tag name')
+@click.pass_context
+def tag_list(ctx, tag_name):
+    """APIC command for listing tagged resources.
+
+    Lists all the resources in ACI with a tagInst.
+    """
+    with ctx.obj['apic_manager'].apic.transaction():
+        # Find all the MOs with the system ID
+        for mo in ctx.obj['apic_manager'].apic.list_mo(
+                ctx.obj['apic_manager'].apic.tagInst.mo, name=tag_name):
```

Comment:
wondering what's the magic here so that we can access the tagInst MO directly thru "ctx.obj['apic_manager'].apic.tagInst.mo".....I couldn't find the magic in the apic_client.py file under apicapi repo.

Reply:
It's using the __getattr__:
https://github.com/noironetworks/apicapi/blob/master/apicapi/apic_client.py#L1074-L1079

Reply:
Ack.